lib/bulbs-elements/store/store.js

```diff
@@ -79,20 +74,4 @@ function dispatchFunction (store, action, payload) {
   }
 
   deliverSubscriptions(store);
-
-  if (DEBUG) {
-    console.groupCollapsed(
-      `DISPATCH %c${action.type} %c=> %c${action.fieldName}`,
-      'color:green',
-      'color:auto',
-      'color:blue'
-    );
-    console.timeEnd(`dispatch ${action.type}`);
-    console.log('PAYLOAD: ', payload);
-    console.log('PREV VALUE: ', priorFieldValue);
-    console.log('NEXT VALUE: ', nextFieldValue);
-    console.log('PREV STORE: ', priorState);
-    console.log('NEXT STORE: ', nextState);
-    console.groupEnd();
-  }
```

Comment:
Where do we draw the line? Using this convention, how would we discourage logging any function call?